homeworks/rostyslav.havuka_RostyslavHavuka/13-ng-forms/forms/src/app/app.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent implements OnInit{
+  form: FormGroup;
+  formEmailValue: string = '';
+  formPassword: string = '';
+  codeEmail: string;
+  codePassword: string;
+  passwordError: string;
```

Comment:
Why do you have so many extra fields? Please. remove them and use local variables for this purpose :smile: